doc/news/9.1.1-vs-9.2.0.h

```diff
@@ -0,0 +1,2579 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 - 2020 by the deal.II authors
```

Comment:
While here, I realize that we might want to change this to 2019 - 2020? Or how do we want this to be?

Reply:
fixed.